### PR TITLE
[ServiceFabric] Update SF app docs. only support for arm deployed resources

### PR DESCRIPTION
**Pull request by a-santamaria**
**Description**

Update the documentation for apps, app types, and services calling out explicitly that the support is only for ARM deployed resources.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
ServiceFabric

File: src/azure-cli/azure/cli/command_modules/servicefabric/_help.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 helps['sf application'] = """
 type: group
-short-summary: Manage applications running on an Azure Service Fabric cluster.
+short-summary: Manage applications running on an Azure Service Fabric cluster. Only supports ARM deployed applications.
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/servicefabric/_help.py:17]:
```suggestion
short-summary: Manage applications running on an Azure Service Fabric cluster. Only support ARM deployed applications.
```

**Review by qwordy — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/servicefabric/_help.py (hunk 1 above)
Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/servicefabric/_help.py:17]: Align verb form with "manage".

Comment (a-santamaria) [reply src/azure-cli/azure/cli/command_modules/servicefabric/_help.py]: Thanks @qwordy, I made the change suggested

**Comment by qwordy** [comment]
Network error, retry.

**Pull request merged**